### DISCUSSION TITLE
Elsa Awesome

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [elsa-workflows/elsa-core#599] [Ideas]
Can we start an Elsa Awesome page covering different aspects of the framework?

e.g. 
- Reusable activities
- Workflow template (WIP, soon)
- Designer custom nodes and links etc. 

Most important

- community projects using Elsa

**Comment by sfmskywalker** [comment]
I love it. Would it be hosted on the documentation site? Or a separate site?

**Comment by GeorgeS2019** [comment]
@sfmskywalker I think the community need to (first) see what categories ARE IN YOUR HEAD right now :-)

We need you combine all the milestones you have in mind into categories and FIRST put them as a page (xxx.md)

Once we have sorted most of the main categories

Then a repository managing the feedback and new proposals will then be necessary.

**Comment by sfmskywalker** [comment]
Not a bad idea 😄 
So before putting things in milestones, describe them first, gather feedback, and based on that, plan.
Are you saying we should separate issues (bugs, questions) in this repo from design/feedback issues into its own repo?

**Comment by GeorgeS2019** [comment]
@sfmskywalker  I think U have lot of experience in Workflow which most of us do not have. I think there is still a lot in your "head" :-)

We help you to organize in ways that will mobilize more developers to come here to, through the Elsa workflow framework you pioneered, into something the .NET community could achieve - especially now, so many different technologies converging. It is difficult to track how best to combine them. 

We iteratively try different formats,  the Elon Musk way, learning each step from each format and moving forwards .

**Comment by sfmskywalker** [comment]
I like the sound of that. Let's do it the Elon Musk way ;-)

How about I just start with a blank issue and write down as much as I can that I have in mind, and we go from there.

**Comment by sfmskywalker** [comment]
I pinned [this issue](468) with everything I have on my mind currently.
